lib/SIL/Utils/SILBridging.cpp

```diff
@@ -291,7 +291,7 @@ BridgedOwnedString BridgedGlobalVar::getDebugDescription() const {
 bool BridgedGlobalVar::canBeInitializedStatically() const {
   SILGlobalVariable *global = getGlobal();
   auto expansion = ResilienceExpansion::Maximal;
-  if (hasPublicVisibility(global->getLinkage()))
+  if (hasPublicVisibility(global->getLinkage(), global->getModule().getOptions().EnableSerializePackage))
```

Comment:
This is wrong: the expansion is maximal for package variables when package cmo is enabled